eokwukwe/ecommerce-shop-api
# Add New Products

#### What does this PR do?
- Admin can add new products 
 #### Description of Task to be completed?
- [x] Admin can add new products functionality
 #### How should this be manually tested?
- clone the branch
- Run `npm install` to install dependencies
- Run `npm run dev` to start the development server
- Log in as an admin to obtain your access token to be sent along in the header as `USER_KEY: Bearer access_token`
- Send a `POST` request to `http://localhost:8000/api/v1/products` with the following credentials:
```
{
  "name": "Chartres Cathedralff",
  "description": "This is a description",
  "price": 10.22 ,
  "discounted_price": 0.00,
  "image": "http://www.image.com",
  "image_2": "http://www.image2.com",
  "thumbnail": "http://www.thumbnail.com",
  "display": 2,
  "category_id": 1,
  "attribute_value_id": 1
}

NOTE: all fields are required
```
- Expected output if the inputs are correct: 
```
{
  "name": "Chartres Cathedralff",
  "description": "This is a description",
  "price": 10.22 ,
  "discounted_price": 0.00,
  "image": "http://www.image.com",
  "image_2": "http://www.image2.com",
  "thumbnail": "http://www.thumbnail.com",
  "display": 2
}
```
 #### What are the relevant pivotal tracker stories?
[#170909404](https://www.pivotaltracker.com/story/show/170909404)
